lib/actions/adapterActions.js

```diff
@@ -1717,16 +1740,18 @@ export function connectToDevice(device) {
     return (dispatch, getState) =>
         new Promise((resolve, reject) => {
             const adapterToUse = getState().app.adapter.bleDriver.adapter;
+            const connectionParams = getState().app.adapter
+                .connectionParameters;
 
             if (adapterToUse === null) {
                 reject(new Error('No adapter selected'));
             }
 
             const connectionParameters = {
-                min_conn_interval: 7.5,
-                max_conn_interval: 7.5,
-                slave_latency: 0,
-                conn_sup_timeout: 4000,
+                min_conn_interval: connectionParams.minConnectionInterval,
+                max_conn_interval: connectionParams.maxConnectionInterval,
+                slave_latency: connectionParams.slaveLatency,
+                conn_sup_timeout: connectionParams.connectionSupervisionTimeout,
```

Comment:
[minor] Instead of creating a new object here with an almost identical name (`connectionParams` and `connectionParameters`), perhaps it's just better to set the `connParams` attribute in `options` directly? 

For example:
```js
const options = {
    scanParams: scanParameters, 
    connParams: {
         min_conn_interval: connectionParams.minConnectionInterval,
         max_conn_interval: connectionParams.maxConnectionInterval,
         slave_latency: connectionParams.slaveLatency,
         conn_sup_timeout: connectionParams.connectionSupervisionTimeout,
    },
};
```